app/models/miq_product_features_share.rb

```diff
@@ -0,0 +1,2 @@
+class MiqProductFeaturesShare < ApplicationRecord
```

Comment:
Just because I have to mention, I still like omitting the model as the convention follows perfectly with HABTM as @carbonin said.


But... Keep it like you have it™ 😄 👍